Create agenda/meeting minutes for 2-11-26 TAC meeting (#132)
PQCA/TAC: meeting-minutes/2026/2026-02-11-TAC.md

**Basil Feedback:**
My time commitment for the mentorship was approximately 2 hours per week throughout the program. This included our weekly sync meetings, reviewing milestones, and providing guidance and technical direction.

I found it very valuable to invest time before the start of the mentorship to define a clear project description and set tangible milestones. Since my mentee is a PhD student who intends to continue developing the project beyond the official mentorship period (e.g., preparing a conference submission or writing a publication), my advice is to view the official project timeline as a guideline for the key milestones, while remaining flexible and open to continued mentoring when appropriate.

Christina also provided an update on blog submissions:
- A mentee blog post is in progress.
- Awaiting David Chisnall’s decision on whether to submit the original or rewritten version of his blog post for PQCA publication.

---

### **Project Updates**
- OQS
  - 
Brian asked about progress on the OQS metric discussion. Norman agreed to revisit the topic at the next OQS TSC meeting and provide updates.

> The team also discussed upcoming OQS release planning, including the need to update mlk-native, which is currently nine months behind the latest drops. Norman noted he will take feedback regarding increased release frequency back to the OQS team and coordinate timing for mldsa-native updates.

Reviewer: ```suggestion
The team also discussed upcoming OQS release planning, including the need to update mlkem-native, which is currently nine months behind the latest drops. Norman noted he will take feedback regarding increased release frequency back to the OQS team and coordinate timing for mldsa-native updates.
```

- PQCP
  - 
Matthias reported on several PQCP updates:

> - Presentation scheduled for March 9 at RWC in Taipei, where Matthias and Hanno will present on mlk-native.

Reviewer: ```suggestion
- Presentation scheduled for March 9 at RWC in Taipei, where Matthias and Hanno will present on mlkem-native.
```

> - Ongoing work on whole-light proofs and extended verification for x86 and ARM.

Reviewer: ```suggestion
- Ongoing work on HOL-Light proofs and extended verification for x86 and ARM.
```

- Continued verification of assembly components.
- Integration and cleanup of an ARM contribution introducing an MVE backend.
- Planned regular time-based releases for both mlk-native and mldsa-native, targeting every 1–3 months.

> Matthias emphasized the importance of aligning mlk-native updates ahead of the RWC talk.

Reviewer: ```suggestion
Matthias emphasized the importance of aligning mlkem-native updates ahead of the RWC talk.
```

- CBOMkit
  - 
Aditya shared updates on CBOM Kit:

> - Added Go language support, now joining Python and Java.

Reviewer: ```suggestion
- Added Go language support on SonarQube, doing integration with CBOMkit
```

> - Go implementation includes support for the standard crypto library and extended X Crypto library on Solar Cube.

Reviewer: ```suggestion
- Go implementation includes support for the standard crypto library and extended X Crypto library on SonarQube.
```

> - No formal roadmap exists for additional languages; however, members from the Engore Foundation expressed interest in contributing support for C and C++, which will be raised as a discussion item.

Reviewer: ```suggestion
- No formal roadmap exists for additional languages; however, members from the NgKore Foundation expressed interest in contributing support for C and C++, which will be raised as a discussion item.
```

- Analyze cryptographic configurations.
- Assess potential quantum risk exposure.

The TAC discussed whether this should be developed as:
- A standalone project under PQCA, or
- An extension to CBOMkit, complementing its existing code-level scanning capabilities.

Aditya and others noted alignment between Dhananjay’s proposal and CBOMkit functionality, suggesting integration may be the more strategic path.

Dhananjay agreed to attend the next CBOMkit meeting to present the proposal in more detail and gather feedback from the CBOMkit TSC members.

---

### **Next Steps / Action Items**

| Action Item | Owner | Status / Due Date |
|------------|-------|------------------|
| Revisit OQS metric discussion at next OQS TSC meeting and report updates | Norman | Next OQS TSC meeting |
| Raise CBOMkit pipeline integration blog post at next CBOM Kit meeting and report back | Aditya | Next TAC meeting |
| Share feedback from previous mentors (Matthias & Basil) with mailing list or include in meeting minutes | Christina | Before next meeting |
| Await David Chisnall’s decision on blog version and, if new, send to content review team | Christina | Pending decision |

> | Take feedback on mlk-native release frequency to OQS team and coordinate release timing | Norman | Before next meeting |

Reviewer: ```suggestion
| Take feedback on mlkem-native release frequency to OQS team and coordinate release timing | Norman | Before next meeting |
```

> | Attend next CBOM Kit meeting to present TLS endpoint scanning proposal | Dhananjay | Next CBOMkit meeting |

Reviewer: ```suggestion
| Attend next CBOMkit meeting to present TLS endpoint scanning proposal | Dhananjay | Next CBOMkit meeting |
```

---

## Voting Representative Attendance (_Alphabetical by 1st name_)
### Premier Member Representatives

* [ ] Brian Jarvis, AWS [TAC Chair]
* [ ] Michael Maximilien, IBM
* [ ] Norman Ashley, Cisco (OQS Rep)
* [ ] Sophie Schmieg, Google

 ### Project Representatives
* [ ] Matthias Kannwischer, TCR
* [ ] Hanno Becker, AWS (PQCP)
* [ ] Andreas Schade, IBM (CBOMkit)

## Non-Voting Representative Attendance

### LF Staff 
* [ ] Christina Harter 
* [ ] Hart Montgomery 
* [ ] Min Yu

### Other Attendees
* [ ] Aditya Koranga, NgKore \[TAC Vice Chair\]

> * [ ] *Enter other attendees*
> 
> ---
> # Meeting Agenda
> - Christina: New Associate Member - NgKore Foundation
> - Christina: Mentorship Program updates
>   - [Call for Mentors](https://github.com/PQCA/TAC/issues/new?template=propose-a-mentorship-project.yml) open until March 13
>   - Feedback to share from previous mentors
>   - Winter Mentorship ending early
>   - Blog post from Basil's fall mentee coming soon
> - Christina: CHERIoT blog cross post: [awaiting approval](https://github.com/CHERIoT-Platform/cheriot-pqc/issues/1) from David Chrisnell
> 

Reviewer 2: ```suggestion
# Meeting Agenda
- Action item follow ups:
  - [Norman] OQS metric
  - [Andreas / Katerina / Finn] CBOMkit Pipeline blog post
- [Christina]: New Associate Member - NgKore Foundation
- [Christina]: Mentorship Program updates
  - [Call for Mentors](https://github.com/PQCA/TAC/issues/new?template=propose-a-mentorship-project.yml) open until March 13
  - Feedback to share from previous mentors
  - Winter Mentorship ending early
  - Blog post from Basil's fall mentee coming soon
- [Christina]: CHERIoT blog cross post: [awaiting approval](https://github.com/CHERIoT-Platform/cheriot-pqc/issues/1) from David Chrisnell

```
